packages/react-dialog/src/stories/AlertDialog.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Button } from '@fluentui/react-button';
+import { Dialog, DialogProps, DialogHeader, DialogBody, DialogFooter } from '../index';
+
+export const AlertDialog = (props: Partial<DialogProps>) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    <>
+      <Button
+        onClick={() => {
+          setIsOpen(true);
+        }}
+        aria-haspopup="true"
+      >
+        Open Dialog
+      </Button>
+
+      <Dialog type="alert" aria-labelledby="dialog-title" open={isOpen} {...props}>
```

Comment:
If Dialog has just a short plain text content, there should be `aria-describedby` pointing to the DialogBody as described in https://www.w3.org/TR/wai-aria-practices/#dialog_modal. We might need to add some guidance regarding this in the best practices and consider adding it to all stories.